### ISSUE TITLE
Qt resources are linked/compiled improperly

### ISSUE THREAD

**Issue by ArsenArsen** [mxe/mxe#1805]
Hello! Recently, I've been trying to embed my icon into my app. The `qrc_*.cpp` file gets generated fine, but after the entire executable is compiled I am left with a missing/broken icon. The result rendered in my program looks like this:  
![](https://i.imgur.com/4rdd7OU.png)

When compiling under Windows natively the icon is just fine:  
![](https://i.imgur.com/0rLbXwh.png)

I have taken the MXE-generated qrc from my CI and applied it in this Windows system, and again it worked just fine, leading me to believe it's an issue with the linker/compiler in MXE.

Thanks for making MXE

**Comment by tonytheodore** [comment]
Are you using a static MXE build? You may need to add `QTPLUGIN += qico` to your *.pro file.

http://doc.qt.io/qt-5/plugins-howto.html#static-plugins

**Comment by tonytheodore** [comment]
It's probably simpler than that, make sure https://github.com/ArsenArsen/KShare/blob/master/icon.qrc and https://github.com/ArsenArsen/KShare/blob/master/mainwindow.ui#L18 match ;)

**Comment by ArsenArsen** [comment]
Yes I am using the static one.
You can ignore ~~and I should update~~ that one, as later on in the code I also: ```cpp
    setWindowIcon(QIcon(":/icons/icon.png"));```

So that shouldn't be the case. I'll try with the plugin. Thanks

**Comment by ArsenArsen** [comment]
Okay, when I tried the plugin, the first thing that happened was: 
```
Project WARNING: Redundant entries in QTPLUGIN: qico
Project WARNING: Redundant entries in QTPLUGIN: qico
Project WARNING: Redundant entries in QTPLUGIN: qico
```
I SCP'd the compiled executable into a shared folder with my VM.

And it worked. Wow. Considering it said it's a redundant entry I tried again, without it. The results were the same. Odd. Sorry for me bothering. But, there is a different issue. The cross compiled version seems to have a few odd bugs which don't happen on the native build (which is now on appveyor)

Thanks again for maintaining this great project. It has saved me a lot of time
